### ISSUE TITLE
loss 

### ISSUE THREAD

**Issue by RitchieHuang11** [andy-yun/pytorch-0.4-yolov3#82]
why loss is so big?

**Comment by andy-yun** [comment]
It depends your data.
After saturation of training steps, it will be small.
